cli/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "validator-bonds-cli"
+version = "0.0.0"
+edition = "2021"
+
+[[bin]]
+name = "validator-bonds-cli"
+path = "src/bin/cli.rs"
+
+[dependencies]
+anyhow = { workspace = true }
+bincode = { workspace = true }
+clap = { workspace = true }
+env_logger = { workspace = true }
+log = { workspace = true }
+merkle-tree = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+serde_yaml = { workspace = true }
+structopt = { workspace = true }
+snapshot-parser = { workspace = true }
+solana-sdk = { workspace = true }
+tokio = { workspace = true }
+anchor-lang = { workspace = true }
+anchor-spl = { workspace = true }
+chrono = { workspace = true }
+solana-clap-utils = { workspace = true }
+solana-client = { workspace = true }
+solana-logger = { workspace = true }
+solana-cli-config = { workspace = true }
+solana-account-decoder = { workspace = true }
+solana-program = { workspace = true }
+rust_decimal = { workspace = true }
+validator-bonds = { version = "0.1.0", path="../programs/validator-bonds", features=["cpi"] }
```

Comment:
maybe this could be done in the same way as @janlegner configured the `merkle-tree` dependency?
https://github.com/marinade-finance/validator-bonds/blob/main/Cargo.toml#L32